src/main/dist/rpm/scripts/postinstall.sh

```diff
@@ -7,9 +7,9 @@ set -e
 case "$1" in
     [1-2])
 
-      Rscript -e "install.packages(c('opal', 'tidyverse', 'knitr', 'rmarkdown'), repos=c('http://cran.rstudio.com', 'http://cran.obiba.org'), dependencies=TRUE, lib='/usr/local/lib/R/site-library')"
+      Rscript -e "install.packages(c('opal', 'tidyverse', 'knitr', 'rmarkdown'), repos=c('http://cran.rstudio.com', 'http://cran.obiba.org'), dependencies=TRUE, lib='/usr/share/R/library')"
```

Comment:
That's not the right location: /usr/share/R/library is for core libraries. The problem exposed in the issue is to guess the "R add-ons" location: sometimes its /usr/local/lib/R/site-library and sometimes it is /usr/lib64/R/library. So you should test for  the existence of these folders and choose the one appropriate for the system.